vg.yaml

```diff
@@ -2,17 +2,18 @@ apiVersion: volumegroup.storage.dell.com/v1
 kind: DellCsiVolumeGroupSnapshot
 metadata:
   # Name must be 13 characters or less in length
-  name: "vg-snaprun1"
+  name: "vg-snaprun52"
   namespace: "helmtest-vxflexos"
 spec:
   # Add fields here
   driverName: "csi-vxflexos.dellemc.com"
   # defines how to process VolumeSnapshot members when volume group snapshot is deleted
   # "Retain" - keep VolumeSnapshot instances
   # "Delete" - delete VolumeSnapshot instances
-  memberReclaimPolicy: "Retain"
-  volumesnapshotclass: "vxflexos-snapclass"
+  memberReclaimPolicy: "Delete"
+  volumesnapshotclass: "vgs-test"
```

Comment:
should we change this SC? Most users will probably have vxflexos-snapclass from the samples/ dir in driver

Reply:
sure 
i will change it